airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/message/DestinationMessage.kt

```diff
@@ -100,9 +135,9 @@ class DestinationMessageFactory(private val catalog: DestinationCatalog) {
                     message.trace.type == AirbyteTraceMessage.Type.STREAM_STATUS &&
                         status.status == AirbyteStreamStatus.COMPLETE
                 ) {
-                    DestinationStreamComplete(stream, message.trace.emittedAt.toLong())
+                    DestinationEndOfStream(stream, message.trace.emittedAt.toLong())
                 } else {
-                    Undefined
```

Comment:
So any trace message other than STREAM_STATUS, COMPLETE becomes StreamIncomplete?

Reply:
good catch. I always forget that there are other statuses >.>